js-intro.js

```diff
@@ -46,6 +74,19 @@ var stringWithVowels2 = "ILoveJavaScript"
 var notAString1 = true
 var notAString2 = 42
 
+var notAString1 = true
+var notAString2 = 42
+
+var noVowelsExtended = (string) => {
```

Comment:
I'm not sure how the user would know that they've made a mistake with their input with this code. If I pass in 42, it returns an empty string. How can you communicate to the user more here?